mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -250,44 +250,40 @@ def _load_mesh(self):  # {{{
             refTopDepth, refLayerThickness
         # }}}
 
-    def _cache_velocity_climatologies(self):  # {{{
+    def _compute_velocity_climatologies(self):  # {{{
         '''compute yearly velocity climatologies and cache them'''
 
-        variableList = ['avgNormalVelocity',
-                        'avgVertVelocityTop']
```

Comment:
I see that. I did a quick search to see if we could retrieve the variable names from a netcdf file using python netcdf (Matlab has this functionality, for example), but I don't believe it is possible.